### DISCUSSION TITLE
Adding PnP.Framework to a windows app

### DISCUSSION THREAD

**Discussion by tsegato** [pnp/pnpframework#149] [General]
When I add the nuget package to a windows app I get following:
Unable to resolve dependency 'PnP.Core'. Source(s) used: 'nuget.org', 'Microsoft Visual Studio Offline Packages'.
Is this by design?

**Comment by jansenbe** [comment]
> When I add the nuget package to a windows app I get following:
> Unable to resolve dependency 'PnP.Core'. Source(s) used: 'nuget.org', 'Microsoft Visual Studio Offline Packages'.
> Is this by design?

That is strange... Do you see the same with for example a console app?

**Reply by peterga28** [reply]
I have the same issue but in Web Project .Net Framework 4.7.2 but for me the missing package is applicationinsights-sdk-labs. Another approach I tried was to install PnP.Core first from nuget and after that the PnP.Framework, without any issue. But I dont want to use PnP.Core if it is still in beta. What is the workaround fo this?

**Reply by jansenbe** [reply]
PnP Framework requires PnP Core...we now that one is still in Beta, but that will change by end of the month.
